apps/aehttp/src/aehttp_api_encoder.erl

```diff
@@ -204,7 +206,8 @@ type2pfx(peer_pubkey)      -> <<"pp">>;
 type2pfx(name)             -> <<"nm">>;
 type2pfx(state)            -> <<"st">>;
 type2pfx(poi)              -> <<"pi">>;
-type2pfx(state_trees)      -> <<"ss">>.
+type2pfx(state_trees)      -> <<"ss">>;
+type2pfx(call_state_tree)  -> <<"cs">>.
```

Comment:
It seems requiring integration of https://github.com/aeternity/protocol/blob/362d566af481214a5f650385b10d6cb3cfd0e831/node/api/api_encoding.md#encoding-scheme-for-api-identifiers-and-byte-arrays